misc/config_template.in.hjson

```diff
@@ -66,7 +66,8 @@
         //  See https://github.com/trentm/node-bunyan#streams
         //
         //  Remember you can pipe logs through Bunyan to pretty-print:
-        //  tail -F ./logs/enigma-bbs.log | bunyan
+        //  Linux   : tail -F ./logs/enigma-bbs.log | bunyan
+        //  Windows : Get-Content .\enigma-bbs.log  | bunyan.cmd
```

Comment:
This should probably read "PowerShell" (which is available in Linux also)

Reply:
Fair enough, Also added tail and -f types.